functions/bot_moderator_function.py

```diff
@@ -1,8 +1,19 @@
+#
+#
+# main() will be run when you invoke this action
```

Comment:
This should probably go above `def main(event):` and not at the top of the file.